libs/shared/typescript/charts-ngx/src/lib/boxplot/boxplot.directive.stories.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  BoxplotProps,
+  CategoryPoint,
+  dynamicBoxplotPoints,
+  staticBoxplotPoints,
+  staticBoxplotSummaries,
+} from '@sagebionetworks/shared/charts';
+import { Meta, StoryObj } from '@storybook/angular';
+import { BoxplotDirective } from './boxplot.directive';
+
+const meta: Meta<BoxplotDirective> = {
+  component: BoxplotDirective,
+  title: 'directives/sageBoxplot',
+  argTypes: {
+    pointTooltipFormatter: {
+      control: { type: 'function' },
+    },
+  },
```

Comment:
Must specify that `pointTooltipFormatter` is a function for Storybook to correctly pass the argument along to the Angular component